package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "reactproject",
+  "version": "1.0.0",
+  "description": "create react project",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "./node_modules/.bin/webpack-dev-server --config ./webpack/webpack.dev.config.js",
+    "flow": "flow"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/vagabondhoang/react-training.git"
+  },
+  "author": "hoang tuan anh",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/vagabondhoang/react-training/issues"
+  },
+  "homepage": "https://github.com/vagabondhoang/react-training#readme",
+  "dependencies": {
+    "babel-core": "^6.26.3",
```

Comment:
từ babel-core xuống đến babel-register là devDependencies not dependencies nhé a

Reply:
done!